Source/Scene/ModelExperimental/FeatureIdPipelineStage.js

```diff
@@ -318,6 +318,13 @@ function processImplicitRange(
   );
```

Comment:
also `SelectedFeatureIdPipelineStage`'s shader should be updated to check for `nullFeatureId`, treating it like the current behavior when the feature ID is greater than the number of features in the property table